bin/sdc-thoth-install

```diff
@@ -92,15 +95,15 @@ if [[ -z $DOWNLOAD_MANTA ]]; then
 
 fi
 
-[[ -n $SDC_ACCOUNT ]] || fatal "must set \$SDC_ACCOUNT. This is the account in manta that stores your thoth data."
-[[ -n $SDC_KEY_ID ]] || fatal "must set \$SDC_KEY_ID. This must already exist in the $SDC_ACCOUNT account, and needs to be available on this node (ssh-agent is recommended)."
+[[ -n $TRITON_ACCOUNT ]] || fatal "must set \$TRITON_ACCOUNT. This is the account in manta that stores your thoth data."
+[[ -n $TRITON_KEY_ID ]] || fatal "must set \$TRITON_KEY_ID. This must already exist in the $TRITON_ACCOUNT account, and needs to be available on this node (ssh-agent is recommended)."
 
-if [[ -z $SDC_URL ]]; then
+if [[ -z $TRITON_URL ]]; then
 	server=us-west-1.api.joyentcloud.com
```

Comment:
Can you please also change this to `joyent.com`? We've stopped maintaining (and redirecting) `joyentcloud.com`.